### DISCUSSION TITLE
Access to logger in extensions

### DISCUSSION THREAD

**Discussion by moeriki** [directus/directus#7737] [Feature Requests]
In custom endpoints and hooks it would be easy to pass the logger as part of the `ExtensionContext`.

You could opt to make a child logger to differentiate Directus logs from application logs.

If you agree I can PR.

FYI this also works now.

```js
const logger = require('directus/dist/logger').default;
logger.info('Hello!');
```

**Comment by rijkvanzanten** [comment]
Agreed! Makes sense to me 👍🏻 

(cc @nickrum)

**Comment by nickrum** [comment]
This has since been added 👍
